format/index.md

```diff
@@ -955,10 +955,86 @@ COMMA, ",") list that refers to the name(s) of the fields to be returned.
 If a client requests a restricted set of [fields], an endpoint **MUST NOT**
 include additional [fields] in the response.
 
+Basic request:
+
+```http
+GET /articles?include=author
+```
+
+```text
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "data": {
+    "type": "articles",
+    "id": "1",
+    "attributes": {
+      "title": "JSON API paints my bikeshed!",
+      "body": "The shortest article. Ever.",
+      "created": 1432306588,
+      "updated": 1432306589
+    },
+    "relationships": {
+      "author": {
+        "linkage": {
+          "id": 42, "type": "people"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "type": "people",
+      "id": 42,
+      "attributes": {
+        "name": "John",
+        "age": 80,
+        "gender": "male"
+      }
+    }
+  ]
+}
+```
+
+Request with `fields` parameter:
+
 ```http
 GET /articles?include=author&fields[articles]=title,body&fields[people]=name
```

Comment:
Relationships must also be included in `fields`, even if they're specified via `include`. Therefore, the `author` field would have to be included in `fields[articles]` to get the response below.

In other words, the request would need to be:

```http
GET /articles?include=author&fields[articles]=title,body,author&fields[people]=name
```